### ISSUE TITLE
爱可可微博热门分享(1.31)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#61]
No 1. <a href="https://weibo.com/1402400261/NElwQhC1c">Hugging Face Datasets(huggingface.co/datasets)上共享数据集规模突破十万，涵盖了艺术、医学、生物学、法律、金融、音乐、化学、气候等多个领域，可以通过简单的搜索和过滤功能找到所需的数据集，可以按任务、大小、子任务、语言、许可证等进行分类和过滤 ......</a>
No 2. <a href="https://weibo.com/1402400261/NEdDQ4kdo">用Langchain, ColBERT, Ragatouille 和 ChromaDB构建高级 RAG 系统，有效地提取 Mixtral 8x7B 生成高质量答案所需的所有相关上下文</a>
No 3. <a href="https://weibo.com/1402400261/NEpuZ4tOu">高质量的GPU编程/架构资源</a>
No 4. <a href="https://weibo.com/1402400261/NEpIy2InI">史料RAG：用向量数据库基于RAG(检索增强生成)方式搭建一个中国历史问答应用，接受用户的询问，从历史语料库中检索相关的历史资料片段，利用大语言模型给出较为可靠的回答</a>
No 5. <a href="https://weibo.com/1402400261/NEpr1orIy">Phantom：通用且自动化的图像增强和超分辨率框架，可用于各种场景和质量的图像，用户只需上传一张低质量的图片，即可一键生成一张经过增强的图片，无需选择 SR 模型或调整参数</a>
No 6. <a href="https://weibo.com/1402400261/NElpEnJ7E">Code Llama 70B在线体验，Code Llama 70B由Meta推出的最新AI编程模型，标志着AI辅助代码生成领域的重大升级，用于处理更复杂的编程任务，具有更高的准确性和效率</a>
No 7. <a href="https://weibo.com/1402400261/NEq0544yM">自主Agent(LLM)相关论文列表</a>
No 8. <a href="https://weibo.com/1402400261/NElCKorQ5">用Proxy-Tuning改进语言模型</a>
No 9. <a href="https://weibo.com/1402400261/NEfZgfdRH">PyrateLimiter：使用漏桶算法的请求限速器</a>
No 10. <a href="https://weibo.com/1402400261/NEpJYbAQM">AI-视频裁剪器是一个功能强大的工具，旨在通过对视频进行智能分析，提取最吸引人的片段并裁剪它们来帮助内容创作者、播客爱好者和视频爱好者轻松地从视频中提取引人入胜的部分，该工具利用先进的语言模型 GPT-4，能够智能地分析视频文字记录，识别最受欢迎和最激动人心的时刻</a>
No 11. <a href="https://weibo.com/1402400261/NEg8IEntJ">Inference-Time Training: 用于长文本生成任务的推理时训练方法，与传统方法不同，它不再依赖 KV 缓存来存储所有上下文信息，而是将这些信息直接嵌入到模型参数中</a>
No 12. <a href="https://weibo.com/1402400261/NEpwaqM7u">LLMs in Five Formulas：关于LLM的直觉教程</a>
No 13. <a href="https://weibo.com/1402400261/NEkvGzOqE">[CV]《MoE-LLaVA: Mixture of Experts for Large Vision-Language Models》</a>
No 14. <a href="https://weibo.com/1402400261/NEdNtgIEy">FinSight：专为投资组合经理、投资者和金融爱好者设计的尖端 AI 助手，利用 GPT-4 和财务数据，FinSight 提供关于公司的深入见解和可操作的总结，帮助做出更明智的投资决策。FinSight 可以帮助用户深入了解从损益表、资产负债表和现金流量表中提取的核心财务指标</a>
No 15. <a href="https://weibo.com/1402400261/NEcQsBmdA">《⁡⁣⁢⁣⁡‍‌⁡⁣⁤​﻿﻿⁢‌⁢⁤‍⁢⁣​‌⁤​⁣⁣⁣﻿‌﻿​‌‍‌​⁤⁣⁣⁢⁢互联网知识地图最全合集（2024 新版） - 飞书云文档》</a>
No 16. <a href="https://weibo.com/1402400261/NEkJejUlf">[CL]《Scaling Sparse Fine-Tuning to Large Language Models》</a>
No 17. <a href="https://weibo.com/1402400261/NEpXWnnXm">图归约(包括图粗化、图浓缩、图稀疏化、图摘要等)相关论文资源列表</a>
No 18. <a href="https://weibo.com/1402400261/NEnBUCrug">WhisperKit：一个Swift开发包，它、将 OpenAI流行的Whisper语音识别模型与Apple的CoreML框架集成在一起，可在 Apple 设备上进行高效的本地推理，可通过Swift软件包管理器集成到 Swift 项目中</a>
No 19. <a href="https://weibo.com/1402400261/NEkuj2fPt">早！[太阳] ​</a>
No 20. <a href="https://weibo.com/1402400261/NEg9W2Mz8">几篇论文实现代码</a>
No 21. <a href="https://weibo.com/1402400261/NEpUVkBMM">OpenAOE：基于 OpenAI 的聊天开源框架...</a>
No 22. <a href="https://weibo.com/1402400261/NEkYVrBsK">今日推介(第1301期)：视觉专家混合路由器实证研究、大型视觉-语言模型的专家混合模型、计算和数据高效的语言建模方法、用迭代数据平滑减轻RLHF中的奖励过拟合和过度优化、将稀疏微调扩展到大型语言模型 ......</a>
No 23. <a href="https://weibo.com/1402400261/NEpq7s0Jp">fabric：开放的AI生态系统，用人工智能增强人类能力</a>
No 24. <a href="https://weibo.com/1402400261/NEkURiQyA">[CV] Divide and Conquer: Language Models can Plan and Self-Correct ......</a>
No 25. <a href="https://weibo.com/1402400261/NEc3W4sOS">Meta发布Code Llama 70B，用于代码生成的LLM的新的、性能更高的版本，与之前的 Code Llama ......</a>
No 26. <a href="https://weibo.com/1402400261/NDhiLmXyi">机器学习/深度学习推荐书单，包括作者自己的一本书《...</a>
No 27. <a href="https://weibo.com/1402400261/NDAonkgKF">非母语者英语科研写作的7个技巧</a>
No 28. <a href="https://weibo.com/1402400261/ND82mm1pV">《单卡 3 小时训练专属大模型 Agent：基于 LLaMA Factory 实战 - 知乎》</a>
No 29. <a href="https://weibo.com/1402400261/NEkFPw1mx">[LG]《Iterative Data Smoothing: Mitigating Reward Overfitting and Overoptimization in RLHF》</a>
No 30. <a href="https://weibo.com/1402400261/NEkSlBsOh">[CV] Object-Driven One-Shot Fine-tuning of Text-to-Image ......</a>
No 31. <a href="https://weibo.com/1402400261/NEptD9fTT">AutoMLOps：生成、配置和部署 CI/CD 集成 MLOps 管道的服务，它弥合了数据科学与 DevOps 之间的鸿沟。AutoMLOps 提供了一个可重复的过程，极大地减少了构建 MLOps 管道所需的时间</a>
No 32. <a href="https://weibo.com/1402400261/NEkQryxH5">[CV] InternLM-XComposer2: Mastering Free-form Text-Image ......</a>
No 33. <a href="https://weibo.com/1402400261/NEkTFrvJU">[CV] Overcoming the Pitfalls of Vision-Language Model Finetuning ......</a>
No 34. <a href="https://weibo.com/1402400261/NEdEV8Uvo">Gemini-OpenAI-Proxy：将 OpenAI API 协议转换成谷歌 Gemini Pro 协议，可以将 OpenAI 的功能无缝集成到使用 Gemini Pro 协议的应用中</a>
No 35. <a href="https://weibo.com/1402400261/NEcDjaIxV">Martian发布大模型推理成本排行榜：不同供应商的大模型服务成本存在5倍以上的差异，吞吐量差异超过6倍，API的选择对最终性能至关重要</a>
No 36. <a href="https://weibo.com/1402400261/NEhKp95x1">晚安～ [月亮] ​</a>
No 37. <a href="https://weibo.com/1402400261/NEpSC0POI">SoniTranslate：视频同步翻译工具</a>
No 38. <a href="https://weibo.com/1402400261/NCHgWxazx">#转发赠书#携手@博文视点Broadview 送出5本《Chatbot从0到1（第2版）：对话式交互实践指南》</a>
No 39. <a href="https://weibo.com/1402400261/NEdKwBuLD">数据驱动无人驾驶相关资源大列表</a>
No 40. <a href="https://weibo.com/1402400261/ND7LfwWfd">从零开始实现LoRA</a>
No 41. <a href="https://weibo.com/1402400261/NEdDhCzXF">ChaCha(儿童情感意识的聊天机器人)：一个利用大语言模型帮助儿童增强情感意识的聊天机器人。由韩国 Naver 人工智能实验室研发，旨在帮助儿童分享他们关于个人事件的情感，ChaCha 使用 GPT-3.5 和 GPT-4 模型来处理儿童的语言输入，并生成相关的提示和反馈</a>
No 42. <a href="https://weibo.com/1402400261/NDLqvpN7X">免费书稿：《机器学习工程》，提供了一系列关于机器学习工程的深入内容，包括但不限于数据处理、模型训练、模型评估、模型部署等方面的知识，可以帮助读者更好地理解和掌握机器学习工程的各个方面，从而在实际工作中更有效地应用机器学习技术</a>
No 43. <a href="https://weibo.com/1402400261/NEaUiuH9E">[LG]《Learning Universal Predictors》</a>
No 44. <a href="https://weibo.com/1402400261/NEkNHxqzr">[LG] The Case for Co-Designing Model Architectures with Hardware ......</a>
No 45. <a href="https://weibo.com/1402400261/NDMIMoa9h">WhisperFusion：具有超低延迟无缝对话功能的AI系统，基于WhisperLive和WhisperSpeech的能力，集成了Mistral，一个大型语言模型(LLM)， 在实时语音转文本管道上运行。LLM和Whisper都经过优化，以TensorRT引擎的形式高效运行，以实现高性能和实时处理能力</a>
No 46. <a href="https://weibo.com/1402400261/NE2zCdeia">《Paul Graham 2006-2023 精选文集（中英对照）》</a>
No 47. <a href="https://weibo.com/1402400261/NEkv3hUZV">[CV]《Routers in Vision Mixture of Experts: An Empirical Study》</a>
No 48. <a href="https://weibo.com/1402400261/NEkFSaDBs">提出一种改进的奖励学习算法IDS，通过迭代数据平滑的方式解决了强化学习中的奖励过拟合和过度优化问题，取得了优异的性能。//[LG]《Iterative Data Smoothing: Mitigating Reward Overfitting and Overoptimization in RLHF》</a>
No 49. <a href="https://weibo.com/1402400261/NDN1hwjwy">Pipelight：自托管的轻量级CLI工具，用于创建微小的自动化流水线，可将CI/CD引入最小的项目，基于Rust，占用空间小(13Mb)并使用TypeScript、Toml和Yaml定义流水线 </a>
No 50. <a href="https://weibo.com/1402400261/NE4KZE8PU">ProChat：快速搭建大语言模型 Chat 对话的前端组件库，旨在赋予开发人员轻松打造丰富、动态和直观的聊天界面的能力</a>
